src/packages/ganache/api-extractor.json

```diff
@@ -79,6 +79,7 @@
     "@ethereumjs/blockchain",
     "@ethereumjs/trie",
     "@ethereumjs/ethash",
+    "@ethereumjs/rlp",
```

Comment:
I don't know if we need this (or some of the other) `@ethereumjs` packages, but I thought I'd be thorough. Do we think I should generate types one by one with each of these to see what's actually needed?